src/components/fundamentals/icons/buildings-icon/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react"
+import IconProps from "../types/icon-type"
+
+const BuildingsIcon: React.FC<IconProps> = ({
+  size = "20",
+  color = "currentColor",
+  ...attributes
+}) => {
+  return (
+    <svg
+      width={size}
+      height={size}
+      viewBox={`0 0 ${size} ${size}`}
```

Comment:
question: won't this result in the icon clipping on some sizes? Usually, we just leave the `viewBox` as is.

Reply:
Probably. I don't know why I changed the `viewBox` here and not on other icons I added 😅